### DISCUSSION TITLE
Can't login after fresh install

### DISCUSSION THREAD

**Discussion by sternburg161** [umami-software/umami#1110] [Q&A] [answered]
Hey together,

I just deployed an umami instance on vercel with a SQL-Server hosted at my place.

The Deployment on Vercel ran just fine, Configuring the SQL-Database was also an easy task since the sql-schema provided everything i needed. However, after Deploying with Database and SALT env, I was not able to login with admin/admin, umami/umami or something like that. So I decided to alter the Password in the Database to a custom one, but i can't get umami to login into with those Credentials. I can copy the string straight out of the database but I'm not able to login. What could be the cause of that?

**Comment by mikecao** [comment, marked as answer]
Close, the login is admin/umami. The password need to be an bcrypt encrypted string, it can't be just a plain string.

**Reply by sternburg161** [reply]
Ohh so https://github.com/mikecao/umami/blob/master/sql/schema.mysql.sql does provide the encrypted string for "umami"? If that's the case, do i also need a specific SALT before logging in or does that not even play a Role in that case?

Thanks for the fast reply! :)

**Reply by mikecao** [reply]
SALT doesn't matter for login. It's only used to generate unique session ids.

**Reply by sternburg161** [reply]
Thank you! That solved it. :)
